packages/buckets/src/buckets.spec.ts

```diff
@@ -477,9 +490,13 @@ describe('Buckets...', function () {
 
       const { root } = await bobBuckets.listPath(rootKey, '')
       try {
-        // Defaults to highwatermark of CHUNK_SIZE
-        const stream = createReadStream(path.join(pth, 'file2.jpg'))
-        await bobBuckets.pushPath(rootKey, 'path/to/bobby.jpg', stream, { root })
+        // Use default highwatermark of CHUNK_SIZE!
+        const stream = fs.createReadStream(path.join(pth, 'file2.jpg'), {
```

Comment:
Revert to using default createReadStream from node. We don't export a helper function anymore, it wasn't worth the extra deps.